### PR TITLE
Feature: Allow to import databases different than the default  one - db-import command

### PR DESCRIPTION
**Pull request by rigoucr**
This PR provide a new option to allow users to import databases different than the default one.

## Issue Reference

Issue Number: #<id>

## Any additional notes/comments?

### CODE REVIEW

File: src/App/Commands/DbImportCommand.php [hunk 1]
```diff
@@ -23,27 +24,32 @@ protected function configure()
         $this->setName('db-import')
             ->setDescription('Executes db import command')
             ->setHelp('Import given db into default database.')
-            ->addArgument('filepath', InputArgument::OPTIONAL, 'File to import');
+            ->addArgument('filepath', InputArgument::OPTIONAL, 'File to import')
+            ->addOption('dbname', NULL, InputOption::VALUE_OPTIONAL, 'The database in that we want to import the data');
```
Comment (kporras07) [inline src/App/Commands/DbImportCommand.php]: The database that you want to import the data in

Comment (kporras07) [inline src/App/Commands/DbImportCommand.php]: lowercase null please

File: src/App/Commands/DbImportCommand.php [hunk 2]
```diff
@@ -23,27 +24,32 @@ protected function configure()
...
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->setupEnv();
 
         $filepath = $input->getArgument('filepath');
-
+        $databaseName = $_SERVER['MYSQL_DATABASE'];
+        if ($optionalDatabaseName = $input->getOption('dbname')) {
+          $databaseName = $optionalDatabaseName;
+        }
```
Comment (kporras07) [inline src/App/Commands/DbImportCommand.php]: `$databaseName = $input->getOption('dbname') ?? $_SERVER['MYSQL_DATABASE'];`

Please try before making the code change but I think this should work

**Review by kporras07 — CHANGES_REQUESTED** [review]
A couple of minor changes but overall looks good!

File: src/App/Commands/DbImportCommand.php (hunk 2 above)
Comment (rigoucr) [reply src/App/Commands/DbImportCommand.php]: Thx, you right , that's better

File: src/App/Commands/DbImportCommand.php (hunk 1 above)
Comment (rigoucr) [reply src/App/Commands/DbImportCommand.php]: Thx

Comment (rigoucr) [reply src/App/Commands/DbImportCommand.php]: Thx

**Review by kporras07 — APPROVED** [review]
LGTM!

**Pull request merged**